### ISSUE TITLE
Answerline Underlining/Bolding

### ISSUE THREAD

**Issue by geoffrey-wu** [geoffrey-wu/qb-packet-parser#7]
The packet parser currently does not have a way to capture how an answerline is [underlined](#) and **bolded**. I'm not entirely sure how feasible this even is for pdfs, but for .docx files, [YAPP](https://github.com/alopezlago/YetAnotherPacketParser) is able to get that info. 

- It is potentially possible to convert the pdfs to html files, but that would also have to be explored.

Alternatively, I could write a (potentially ML-based) program that can automatically underline the answerline for me and use quizdb data as reference.

**Comment by alopezlago** [comment]
PDF formatting can be a crapshoot; your best bet is likely something that converts PDF to another format with explicit formatting (HTML, DOCX).

**Comment by geoffrey-wu** [comment]
> PDF formatting can be a crapshoot; your best bet is likely something that converts PDF to another format with explicit formatting (HTML, DOCX).

Yep, I converted the .pdf files to .docx and it worked like a charm. Closed by 03a7cf612b973c0287fc29e1ee6bd51df2b25480
